src/class.js

```diff
@@ -1,11 +1,18 @@
+
 // Create a class called User.
 // The constructor of the class should have a parameter called `options`.
 // `options` will be an object that will have the properties `email` and `password`.
 // Set the `email` and `password` properties on the class.
 // Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
 // for a potential password that will be compared to the `password` property.
 // Return true if the potential password matches the `password` property.  Otherwise return false.
-
+class User {
```

Comment:
This may have had something to do with the file but the indentation is different here than it was in the previous file.  I recommend a tab of 2 spaces.  The main point though is to pick one style and stay consistent with it throughout your project.